src/data/faq_de.json

```diff
@@ -2181,18 +2181,102 @@
+                },
+                {
+                    "term": "Begegnungs-Aufzeichnung",
+                    "anchor": "exposure_detection",
+                    "description": "Liste der empfangenen und vorübergehend im Betriebssystemspeicher abgelegten kurzlebigen zufälligen Bluetooth-IDs. Die Begegnungs-Aufzeichnung wird im Rahmen der Risiko-Ermittlung bei der Risiko-Überprüfung gelesen. Alle kurzlebigen zufälligen Bluetooth-IDs, die älter als 14 Tage sind, werden automatisch gelöscht. Die Begegnungs-Aufzeichnung kann von Personen, die die App nutzen, jederzeit auch als Ganzes gelöscht werden. Bei der Deinstallation der CWA wird die Begegnungs-Aufzeichnung automatisch gelöscht."
+                },
+                {
+                    "term": "Business Rules",
                     "anchor": "business_rules",
-                    "description": "Im Kontext der Einführung <a href='#glossary_digital_certificate'>digitaler Zertifikate</a> für COVID-19 wurde durch das eHealth Network der Europäischen Kommission und durch die EU-Länder abgestimmt, dass jedes Land die individuellen Regeln zusammenstellt, die im grenzüberschreitenden Reiseverkehr zu prüfen sind. Diese „Business Rules“ bestehen in der Regel aus zwei Kategorien: <ul><li>Ungültigkeitsregeln (invalidation rules) des Herausgeber-Landes (country of administration), die bestimmen, unter welchen Umständen ein digitales Zertifikat nicht anerkannt wird</li><li>Akzeptanz-Regeln (acceptance rules) des Ziel-Landes (country of arrival), die erfüllt sein müssen, um in das Land einreisen zu können</li></ul>"
+                    "description": "Im Kontext der Einführung <a href='#glossary_digital_certificate'>digitaler Zertifikate</a> für COVID-19 wurde durch das eHealth Network der Europäischen Kommission und durch die EU-Länder abgestimmt, dass jedes Land die individuellen Regeln zusammenstellt, die im grenzüberschreitenden Reiseverkehr zu prüfen sind. Diese „Business Rules“ bestehen in der Regel aus zwei Kategorien: <ul><li>Ungültigkeitsregeln (invalidation rules) des Herausgeber-Landes (country of administration), die bestimmen, unter welchen Umständen ein <a href='#glossary_digital_certificate'>digitales Zertifikat</a> nicht anerkannt wird</li><li>Akzeptanz-Regeln (acceptance rules) des Ziel-Landes (country of arrival), die erfüllt sein müssen, um in das Land einreisen zu können</li></ul>"
+                }
+            ],
+            "C": [
+                {
+                    "term": "Check-in",
+                    "anchor": "check_in",
+                    "description": "Der Check-in ist ein Verfahren, bei dem eine Person sich bei einem gewissen Ort anmeldet. Um die Check-in-Funktion der CWA für die Kontaktverfolgung anzubieten, ist es notwendig, dass im Eingangsbereich oder beim Zutritt zu einer Veranstaltung ein QR-Code veröffentlicht wird, den die Nutzer der CWA einscannen, um sich digital einzuchecken und als Kontakt erfasst zu werden."

```

Comment:
Ort oder Event 

Die Nutzer, die sich einchecken werden nirgends erfasst.

Reply:
Will be changed in the next commit, thanks!